### ISSUE TITLE
Problem with OSX shortcut under Linux Mint

### ISSUE THREAD

**Issue by ottsch** [ColemanGariety/galileo#10]
```
/usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require': /var/lib/gems/1.9.1/gems/galileo-0.2.0/lib/galileo.rb:119: invalid multibyte char (US-ASCII) (SyntaxError)
/var/lib/gems/1.9.1/gems/galileo-0.2.0/lib/galileo.rb:119: invalid multibyte char (US-ASCII)
/var/lib/gems/1.9.1/gems/galileo-0.2.0/lib/galileo.rb:119: syntax error, unexpected $end, expecting ':'
...tem_name'] == 'OSX' ? 'Link (⌘  + Click)' : 'Link']
...                               ^
    from /usr/lib/ruby/1.9.1/rubygems/custom_require.rb:36:in `require'
    from /var/lib/gems/1.9.1/gems/galileo-0.2.0/bin/galileo:3:in `<top (required)>'
    from /usr/local/bin/galileo:23:in `load'
    from /usr/local/bin/galileo:23:in `<main>'
```

**Comment by ColemanGariety** [comment]
Can you be more specific as to what caused the issue?

**Comment by ColemanGariety** [comment]
Never-mind, this is a duplicate and should be fixed as of 9ef10b3742.
